bigquery/bigquery_reservation_quickstart/main.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START bigqueryreservation_quickstart]
+
+// The bigquery_reservation_quickstart application demonstrates usage of the
+// BigQuery reservation API by enumerating some of the resources that can be
+// associated with a cloud project.
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	bqReservation "cloud.google.com/go/bigquery/reservation/apiv1"
```

Comment:
Package names, even when renamed, should be all lower case. https://golang.org/doc/effective_go.html#package-names

Reply:
done.